css/properties/break-inside.json

```diff
@@ -75,8 +75,24 @@
                 "ie": {
                   "version_added": "10"
                 },
-                "opera": "mirror",
-                "opera_android": "mirror",
+                "opera": [
```

Comment:
@mstensho I confirmed parser support for `break-inside: avoid-column` in Presto and am assuming it was properly supported. Can you confirm?